packages/experiments/src/Text.ts

```diff
@@ -1 +1,2 @@
-export * from './components/Text/index';
+// @deprecated Import from office-ui-fabric-react package instead.
```

Comment:
You _may_ need a full jsdoc block for `@deprecated` to be parsed properly. If so, I recommend modifying this to simply:

```suggestion
/**
 * @deprecated Import from office-ui-fabric-react package instead.
 */
```